sos/report/plugins/nvme.py

```diff
@@ -18,7 +18,11 @@ class Nvme(Plugin, IndependentPlugin):
     packages = ('nvme-cli',)
 
     def setup(self):
-        self.add_copy_spec("/etc/nvme/discovery.conf")
+        self.add_copy_spec([
+            "/etc/nvme/discovery.conf",
+            "/etc/nvme/hostnqn",
+            "/sys/module/nvme_core/parameters/multipath"
```

Comment:
This is already collected by the kernel plugin, along with the rest of `/sys/module/nvme_core/parameters`.